lib/tasks/case_contact_types_reminder.rb

```diff
@@ -0,0 +1,96 @@
+class CaseContactTypesReminder
+  include Rails.application.routes.url_helpers
+
+  NEW_CASE_CONTACT_PAGE_PATH = "https://casavolunteertracking.org/case_contacts/new"
+  FIRST_MESSAGE = "It's been 60 days or more since you've reached out to these members of your youth's network:\n"
+  THIRD_MESSAGE = "If you have made contact with them in the past 60 days, remember to log it: "
+
+  def send!
+    responses = []
+    eligible_volunteers = Volunteer.where(receive_sms_notifications: true)
+      .where.not(phone_number: nil)
+      .select { |v| !last_reminder_within_quarter(v) }
+
+    eligible_volunteers.each do |volunteer|
+      uncontacted_case_contact_type_names = uncontacted_case_contact_types(volunteer)
+      if uncontacted_case_contact_type_names.count > 0
+        responses.push(
+          {
+            volunteer: volunteer,
+            messages: send_sms_messages(volunteer, uncontacted_case_contact_type_names)
+          }
+        )
+        update_reminder_sent_time(volunteer)
+      end
+    end
+
+    responses
+  end
+
+  private
+
+  def uncontacted_case_contact_types(volunteer)
+    contacted_types = volunteer.case_contacts.where("occurred_at > ?", 2.months.ago).joins(:contact_types).pluck(:name)
+    ContactType.all.pluck(:name).uniq - contacted_types
+  end
+
+  def send_sms_messages(volunteer, uncontacted_case_contact_type_names)
+    volunteer_casa_org = volunteer.casa_org
+    if !valid_casa_twilio_creds(volunteer_casa_org)
+      return
+    end
+
+    twilio_service = TwilioService.new(volunteer_casa_org.twilio_api_key_sid, volunteer_casa_org.twilio_api_key_secret, volunteer_casa_org.twilio_account_sid)
+    sms_params = {
+      From: volunteer_casa_org.twilio_phone_number,
+      Body: nil,
+      To: volunteer.phone_number
+    }
+
+    messages = [
+      FIRST_MESSAGE,
+      uncontacted_case_contact_type_names.map { |name| "• #{name}" }.join("\n"),
+      THIRD_MESSAGE + new_case_contact_page_short_link
```

Comment:
iMessage or my provider (Mint mobile) is blocking the message with the short link to be blocked. Twilio reports it as delivered tho 😕 
<img width="1408" alt="Screen Shot 2022-06-06 at 10 06 38 PM" src="https://user-images.githubusercontent.com/47438886/172300194-1cddbf40-58db-4a38-a342-790c169b981c.png">
<img src="https://user-images.githubusercontent.com/47438886/172300339-9df85362-8067-4239-b6a1-dbe5773463df.jpg" width="400" />